packages/client-core/src/world/Location.tsx

```diff
@@ -60,7 +63,25 @@ const LocationPage = ({ offline }: Props) => {
   if (params.locationName) {
     useLoadLocation({ locationName: params.locationName })
   } else {
-    useLoadScene({ projectName: params.projectName!, sceneName: params.sceneName! })
+    let projectId, sceneId
+    if (params.projectName) {
+      Engine.instance.api
+        .service(projectPath)
+        .find({ query: { name: params.projectName, $limit: 1 } })
```

Comment:
we should use the sceneName provided from the `params` to search in the query as well as the project name. we can avoid returning all scenes in the project and searching on the client